data_tools/fsgp_2024_laps.py

```diff
@@ -0,0 +1,144 @@
+import pandas as pd
+from os import path
+from datetime import datetime
+import pytz
+
+
+class FSGPDayLaps:
+
+    def __init__(self, day: int):
+        self.day = day
+
+        self.filepath = path.normpath(path.join(path.dirname(__file__),
```

Comment:
Use `pathlib` with `pathlib.Path(__file__).parent.parent / "fsgp_timing_data" / "fsgp_timing_day_{day}.csv"` for robust platform-independence